test/expect/TestFuser.test_lstm_concat_cuda.expect

```diff
@@ -25,39 +25,52 @@ with prim::FusionGroup_0 = graph(%0 : Float(*, *)
   %13 : Float(*, *), %14 : Float(*, *), %15 : Float(*, *), %16 : Float(*, *) = prim::ConstantChunk[chunks=4, dim=1](%2)
   %17 : Float(*, *), %18 : Float(*, *), %19 : Float(*, *), %20 : Float(*, *) = prim::ConstantChunk[chunks=4, dim=1](%1)
   %21 : int = prim::Constant[value=1]()
-  %22 : Float(*, *) = aten::add(%13, %17, %21)
-  %23 : int = prim::Constant[value=1]()
-  %24 : Float(*, *) = aten::add(%14, %18, %23)
+  %22 : int = prim::Constant[value=1]()
```

Comment:
(observation) we should probably run DCE on fusion group bodies...

Reply:
Sure... I can add that

Reply:
I actually don't have a clue where are those coming from. I guess because we postpone merging of different groups until later?

Reply:
No clue... don't worry about adding it in this PR, we should have it eventually